### PR TITLE
release v0.1.1

### PR DESCRIPTION
**Pull request by trz42**
adding release notes for v0.1.1 and updating `main` branch

### CODE REVIEW

File: RELEASE_NOTES [hunk 1]
```diff
@@ -1,6 +1,20 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.1.1 (14 november 2023)
+--------------------------
+
+This is a minor release of the EESSI build-and-deploy bot.
+
+Bug fixes:
+* omit header in output of squeue run by the job manager (#220)
```
Comment (bedroge) [inline RELEASE_NOTES:10]:
```suggestion
* omit header in output of `squeue` command run by the job manager (#220)
```

**Comment by boegel** [comment]
@trz42 It probably makes more sense to update the release notes in the `develop` branch first, then collapse the `develop` branch into the `main` branch, and then tag the merge commit as the release?

Should be possible to change the target branch in this PR to `develop` without too much trouble I think...

**Comment by trz42** [comment]
> @trz42 It probably makes more sense to update the release notes in the `develop` branch first, then collapse the `develop` branch into the `main` branch, and then tag the merge commit as the release?
> 
> Should be possible to change the target branch in this PR to `develop` without too much trouble I think...

Changed the target branch to `develop`.

File: RELEASE_NOTES [hunk 2]
```diff
@@ -1,6 +1,20 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.1.1 (14 november 2023)
```
Comment (boegel) [inline RELEASE_NOTES:4]: Nitpicking: let's fix this (see also https://www.grammarly.com/blog/capitalization-the-days-of-the-week-the-months/), same for `v0.1.0` release date

```suggestion
v0.1.1 (14 November 2023)

File: RELEASE_NOTES [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+Bug fixes:
+* omit header in output of `squeue` command run by the job manager (#220)
+* make bot compatible with more recent versions of PyGithub (#224)
+
+Improvements:
+* added a script to help reducing disk usage by cleaning up tarballs stored by jobs (#217)
+* clarified documentation about setting permissions and event subscriptions for the bot (#221)
+
+
 v0.1.0 (29 september 2023)
```
Comment (boegel) [inline RELEASE_NOTES:18]:
```suggestion
v0.1.0 (29 September 2023)

File: RELEASE_NOTES [hunk 4]
```diff
@@ -1,6 +1,20 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.1.1 (14 november 2023)
+--------------------------
+
+This is a minor release of the EESSI build-and-deploy bot.
```
Comment (boegel) [inline RELEASE_NOTES:7]: strictly speaking it's a patch release (minor would be v0.2.0, major would be v1.0.0), but I don't want to get too pedantic ;)

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by trz42** [comment]
@boegel addressed your comments too.

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**